### PR TITLE
feat: add business blob limit

### PR DESCRIPTION
**Pull request by darkskygit**

### CODE REVIEW

**Comment by darkskygit** [comment]
Current dependencies on/for this PR:
* `canary`
  * **PR #5734** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/5734?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/toeverything/AFFiNE/5734?utm_source=stack-comment).

File: packages/backend/server/src/core/quota/schema.ts [hunk 1]
```diff
@@ -71,6 +71,25 @@ export const Quotas: Quota[] = [
...
+  {
+    feature: QuotaType.FreePlanV1,
+    type: FeatureKind.Quota,
+    version: 3,
+    configs: {
+      // quota name
+      name: 'Free',
+      // single blob limit 10MB
+      blobLimit: 10 * OneMB,
+      // used for business logic
```
Comment (forehalo) [inline packages/backend/server/src/core/quota/schema.ts]: more clean description about why both `blobLimit` and `bussnessBlobLimit`

Comment (darkskygit) [reply packages/backend/server/src/core/quota/schema.ts]: see [here](https://affine-pro.slack.com/archives/C06CF33SK96/p1704702840175909) and [here](https://affine-pro.slack.com/archives/C06CF33SK96/p1705388074754999?thread_ts=1705377741.451019&cid=C06CF33SK96)
basically this is a product decision, we will use 100m hard limit in server, but will have smaller limit in frontend based on plan

Comment (darkskygit) [reply packages/backend/server/src/core/quota/schema.ts]: some detail about this field has been add to comment

**Review by forehalo — APPROVED** [review]

**Pull request merged**